portsyncd/portsyncd.cpp

```diff
@@ -135,28 +131,6 @@ int main(int argc, char **argv)
 
                     g_init = true;
                 }
-                if (!port_cfg_map.empty())
-                {
-                    handlePortConfig(p, port_cfg_map);
-                }
-            }
-            else if (temps == (Selectable *)&portCfg)
-            {
-                std::deque<KeyOpFieldsValuesTuple> entries;
-                portCfg.pops(entries);
-
-                for (auto entry: entries)
-                {
-                    string key = kfvKey(entry);
-
-                    if (port_cfg_map.find(key) != port_cfg_map.end())
-                    {
-                        /* For now we simply drop previous pending port config */
-                        port_cfg_map.erase(key);
-                    }
-                    port_cfg_map[key] = entry;
-                }
-                handlePortConfig(p, port_cfg_map);
```

Comment:
put a log here to prove that: one field change will trigger all port configuration change